<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1416</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_15">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>36</pointsize>
                  <kerning>false</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Fashion Paradise</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="Bill_Number_Label">
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="whatsThis">
                     <string extracomment="Bill Identification Number"/>
                    </property>
                    <property name="text">
                     <string>Bill No    :     </string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="Billed_By_Label">
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="whatsThis">
                     <string extracomment="Staff who is billing"/>
                    </property>
                    <property name="text">
                     <string>Billed By :                          </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="1">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="Bill_Date_Label">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="whatsThis">
                     <string extracomment="Date of the Bill"/>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="text">
                     <string>Bill Date :                           </string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="Bill_Time_Label">
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="whatsThis">
                     <string extracomment="Time of the Bill"/>
                    </property>
                    <property name="text">
                     <string>Bill Time :                    </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QTableWidget" name="Bill_Table">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>610</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>16</pointsize>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::NoPen</enum>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>128</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>S.No.</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <stylestrategy>PreferAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
               <property name="foreground">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Item ID</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>true</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Item Name </string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Rate</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Quantity</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Discount (%)</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Discount (Rs.)</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Price (Rs.)</string>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <bold>true</bold>
                 <kerning>false</kerning>
                </font>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>17</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="4" column="0">
                 <widget class="QPushButton" name="Print_Button">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>14</pointsize>
                    <kerning>false</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>Print Bill</string>
                  </property>
                  <property name="icon">
                   <iconset theme="printer"/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <widget class="QLabel" name="Net_Total_Label">
                    <property name="minimumSize">
                     <size>
                      <width>200</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <bold>true</bold>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="text">
                     <string>Net Total          :      </string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="Net_Discount_Label">
                    <property name="minimumSize">
                     <size>
                      <width>200</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Times New Roman</family>
                      <pointsize>16</pointsize>
                      <bold>true</bold>
                      <kerning>false</kerning>
                     </font>
                    </property>
                    <property name="text">
                     <string>Net Discount    :      </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="Total_Label">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>16</pointsize>
                    <bold>true</bold>
                    <kerning>false</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>Total                  :         </string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Minimum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string> </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="My_Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1416</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="New_Bill_Tab">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuStock">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Stock</string>
     </property>
     <addaction name="Stock_Menu"/>
     <addaction name="Sales_Menu"/>
    </widget>
    <widget class="QMenu" name="menuSales">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Sales</string>
     </property>
     <addaction name="actionToday_s_Sales"/>
     <addaction name="actionPrevious_Sales"/>
    </widget>
    <addaction name="menuStock"/>
    <addaction name="menuSales"/>
    <addaction name="Staff_Management_Menu"/>
   </widget>
   <widget class="QMenu" name="Quit_Tab">
    <property name="title">
     <string>Account </string>
    </property>
    <addaction name="Profile"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="New_Bill_Tab"/>
   <addaction name="Quit_Tab"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProfile">
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="Stock_Menu">
   <property name="text">
    <string>View Stock</string>
   </property>
  </action>
  <action name="Sales_Menu">
   <property name="text">
    <string>Update Stock</string>
   </property>
  </action>
  <action name="actionToday_s_Sales">
   <property name="text">
    <string>Today's Sales</string>
   </property>
  </action>
  <action name="actionPrevious_Sales">
   <property name="text">
    <string>Previous Sales</string>
   </property>
  </action>
  <action name="Staff_Management_Menu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Staff Management </string>
   </property>
  </action>
  <action name="Profile">
   <property name="text">
    <string>Profile </string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout and Exit</string>
   </property>
  </action>
  <action name="Exit_Button">
   <property name="text">
    <string>Logout and Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Qt resources/Report_Manager/image.qrc"/>
  <include location="BMS.qrc"/>
 </resources>
 <connections/>
</ui>
